engineering/documents/README.tmpl.md

```diff
@@ -0,0 +1,89 @@
+![logo if any exists](path/to/logo)
```

Comment:
Adding [comment](https://github.com/src-d/guide/pull/84#issuecomment-356618854) from @bzz here to keep track of it

> Guys after reading README template, I have few questions:
> 
> Do we really expect to have Example of utilization section or do we expect each project to re-phrase it to something like Usage or Run, etc?
> As a developer, I find it very useful when readme includes a section on Development with at least a high-level description on how to build/test the project. As many of our projects are for developers, do you think it might be worth adding such thing near to the end? (With many be a link to CONTRIBUTING.md for excessive details)
> As a user, I find it very useful to have a Quickstart of TL;DR style of section that describes main usage in few lines, so a fist-time users can copy and run them immediately to accommodate their goal of trying out new things very quickly, before spending more time on reading/exploring it. Would that be something useful to have?
> One more thing that I find really useful - explicit statement about pre-requisites or runtime dependencies or assumptions about environement that the project has in order to be operational. Ideally those would have been incapsulated in Docker images, so a commands to start DB, a Queue, a Bblfsh, etc might be given, or, at worst, links to each dependency documentation can be provided. So user does not need to discover this though trail and errors, on trying to kickstart the process. What do you think?
>

Reply:
1. I think "Example of utilization" is good, but not necessarily the best. So up to the maintainer of the project to use whatever they feel like.

2. I think the build/test instructions could be part of the "Example of utilization" section if it's simple enough. If not I'd probably simply have a link to "building from source code". Most people will use Docker or aptitude packages when available.

3. This is exactly the kind of context I expect to see in the "Example of utilization" for tools. Something like `make build; cmd foo bar`

4. I would expect these requirements to be part of the installation instructions yeah. See line 46 for this. Maybe I should make it more obvious?

Reply:
2. Sounds great. Do you think it may be worth to articulate that somehow [in there](https://github.com/src-d/guide/pull/84/files#diff-7b3589b30d846038e6c830ddcc5a25daR51) that it's acceptable to have short build commands in that section?

4. From my experience, indeed it's totally worth to go extra mile here and make it painfully obvious